src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -148,7 +148,7 @@ export default [
             const isTaskAction = ReportActionsUtils.isTaskAction(reportAction);
             const isWhisperAction = ReportActionsUtils.isWhisperAction(reportAction);
             return (
-                !isWhisperAction &&
+                (!isWhisperAction || reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU || isReportPreviewAction) &&
```

Comment:
Wondering if `reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU` should be `isIOUAction` instead, what do you think?